docs/src/pages/components/app-bar/app-bar-ru.md

```diff
@@ -11,35 +11,35 @@ components: AppBar, Toolbar, Menu
 
 Может трансформироваться в контекстную панель действий или использоваться в качестве панели навигации.
 
-## Simple App Bar
+## Простая панель навигации
 
-{{"demo": "pages/components/app-bar/ButtonAppBar.js"}}
+{{"demo": "pages/demos/app-bar/ButtonAppBar.js"}}
```

Comment:
Strange

Reply:
Misapplied old translation by the looks of it. Fixed (in Crowdin). We can do it manually here.